src/EPR.Calculator.Frontend/Program.cs

```diff
@@ -36,6 +39,10 @@
     // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
     app.UseHsts();
 }
+else
+{
+    app.UseExceptionHandler(_ => { });
```

Comment:
Please remove blank arrow function

Reply:
@udaydefra , For adding the ExceptionHandlerMiddleware to the request pipeline, blank function was giving error but with using error page path the adding is successful